src/ui/labelWithButton/LabelWithButton.test.tsx

```diff
@@ -0,0 +1,24 @@
+import { theme } from 'assets/styles/theme.styles';
+import { render } from 'tests';
+import { LabelWithButton } from './LabelWithButton';
+
+describe('LabelWithButton component', () => {
+  it('renders label correctly', () => {
+    const { getByText } = render(<LabelWithButton text={'new'} prefix={'condtion'} onClick={() => {}} />);
+
+    expect(getByText('new')).toBeInTheDocument();
+    expect(getByText('condtion:')).toBeInTheDocument();
+    expect(getByText('condtion:')).toHaveStyle({ color: theme.color.text.grey });
+  });
+
+  const mockFn = jest.fn();
+
+  it('renders clickable button', () => {
```

Comment:
Tutaj ta nazwa nie jest najszczęśliwsza. Przy sidebarze miałeś podobny scenariusz sprawdzania odpalania callbacka. Jak to inaczej nazwać?

Reply:
Może `handles clicks correctly`